src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/DynamicRevocationTests.cs

```diff
@@ -1266,105 +1246,6 @@ public static void CheckIntermediateWithExpiredRevocation(PkiOptions pkiOptions)
             }
         }
 
-        internal static void BuildPrivatePki(
-            PkiOptions pkiOptions,
-            out RevocationResponder responder,
-            out CertificateAuthority rootAuthority,
-            out CertificateAuthority intermediateAuthority,
-            out X509Certificate2 endEntityCert,
-            [CallerMemberName] string testName = null,
-            bool registerAuthorities = true,
-            bool pkiOptionsInSubject = false)
-        {
```

Comment:
Please leave a method here with this signature (and current callers calling it), even if it's just deferring to the version now in CertificateAuthority.  I'm anticipating that CertificateAuthority's version will become more generalized over time, and this one will just call that one with the correct arguments (and requirements) to fulfill the needs of the tests here.

Similar to the helper for SslStream's tests, but with a much more complex signature :smile:.

Reply:
updated. I agree that CertificateAuthority will likely grow.